app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative 'lib/database_connection'
+require_relative 'lib/item_repository'
+require_relative 'lib/order_repository'
+require 'date'
+
+class Application
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
+  private
+
```

Comment:
I like how you split this up in to their own methods. Makes sense logically and also looks good.